examples/src/shader_f16/shader.wgsl

```diff
@@ -0,0 +1,7 @@
+@group(0) @binding(0)
```

Comment:
Currently works without enable directive, TODO.

Reply:
Resolved in [34c9ef9](https://github.com/gfx-rs/wgpu/pull/5701/commits/34c9ef9c13798d85e5eb1cc5b98aef762a7b336d)

Suggestions on a more elegant way to do it are encouraged.